test/GRAPHFILES

```diff
@@ -288,7 +274,37 @@ users/npadmana/twopt/twopt-buildtrees.graph
 users/npadmana/twopt/twopt-paircount.graph
 # suite: Elegance
 studies/elegance/elegance.graph
-# suite: Math operations
+# suite: Scalar Performance
```

Comment:
You had it right: "scalar" was said in the meeting, but I think "serial" is a better title.